src/VerifyDeletionModal/VerifyDeletionModal.js

```diff
@@ -24,6 +26,9 @@ const VerifyDeletionModal = ({ setVerifyDeletion, deleteItem, closeModal, catego
       <div className='modal-buttons'>
         <button className='modal-button' onClick={() => {
           determineItemToDelete()
+          const deletionData = compileDeletionData()
```

Comment:
This is actually supposed to be part of `determineItemToDelete`. We need to work through this.